doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,106 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
+===================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps in this section.
+
+The following procure for creating a Fluent Docker container uses a local
+Ubuntu machine to copy the needed files from the Fluent
+
+
+Requirements
+============
+
+* A Linux machine, preferably with Ubuntu 18.04 or later.
+  CentOS Linux distribution is not supported anymore.
+  This machine needs to have `Docker <https://www.docker.com>`_ installed.
+
+* A valid Ansys account. Your Ansys reseller should have
+  provide you with one.
+
+* The following provided files:
+  
+  * `Dockerfile <https://github.com/ansys/pyfluent/blob/main/docker/fluent/Dockerfile>`_
+  * `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_
+
+
+Procedure
+=========
+
+If you have cloned `PyFluent <https://github.com/ansys/pyfluent>`_ locally then change current working directory to
+`Docker files <https://github.com/ansys/pyfluent/blob/main/docker/fluent>`_ before executing these commands
+otherwise copy these files in an empty folder and execute these commands from that folder.
+
+Specify the Ansys installation directory
+----------------------------------------
+
+Specify the pre-installed Ansys directory as a command line argument.
+
+.. code:: python
+
+    python copy_docker_files.py <path to 'ansys_inc' directory>
+
+Copy needed files
+-----------------
+
+Run this script to copy needed files from the Ansys installation directory
+to the container.
+
+.. code:: python
+
+    python copy_docker_files.py <path to 'ansys_inc' directory>
+
+Not all the installation files are copied. These files indicate the files that are
+ignored during the copying:
+
+  * `excludeCEIList.txt <https://github.com/ansys/pyfluent/blob/main/docker/fluent/excludeCEIList.txt>`_
+  * `excludeFluentList.txt <https://github.com/ansys/pyfluent/blob/main/docker/fluent/excludeFluentList.txt>`_
```

Comment:
We could add some more information here:
1. We have excluded these files because we have determined that they are not needed to run typical Fluent workflows.
2. If you find that some of the excluded files are need to run your workflows then you can remove those files from the exclusion list and repeat the build process to create a new image.

Reply:
Done. Thank you.